### ISSUE TITLE
Invalid memory address panic

### ISSUE THREAD

**Issue by TcNobo** [j6s/mailcow-exporter#11]
It was working fine earlier (multiple hours), but all of a sudden, it's giving me this error. 

```
2022/08/13 03:03:25 https://127.0.0.1:7080/api/v1/get/mailq/all
2022/08/13 03:03:25 http: panic serving 172.23.0.1:37478: runtime error: invalid memory address or nil pointer dereference
goroutine 34 [running]:
net/http.(*conn).serve.func1(0xc000208000)
	/usr/local/go/src/net/http/server.go:1801 +0x147
panic(0x8adee0, 0xc5e2e0)
	/usr/local/go/src/runtime/panic.go:975 +0x47a
github.com/j6s/mailcow-exporter/mailcowApi.MailcowApiClient.Get(0x92c264, 0x5, 0xc00021c040, 0xe, 0xc000228014, 0x22, 0xc0002042a0, 0xc0002043c0, 0xc0002044b0, 0x93185e, ...)
	/build/mailcowApi/client.go:72 +0x503
github.com/j6s/mailcow-exporter/provider.Mailq.Provide(0x92c264, 0x5, 0xc00021c040, 0xe, 0xc000228014, 0x22, 0xc0002042a0, 0xc0002043c0, 0xc0002044b0, 0x0, ...)
	/build/provider/mailq.go:26 +0x299
main.collectMetrics(0x92c264, 0x5, 0xc00021c040, 0xe, 0xc000228014, 0x22, 0x4cdded)
	/build/main.go:53 +0x51b
main.main.func1(0x9bdd00, 0xc000238000, 0xc000226000)
	/build/main.go:105 +0x1ef
net/http.HandlerFunc.ServeHTTP(0x94c688, 0x9bdd00, 0xc000238000, 0xc000226000)
	/usr/local/go/src/net/http/server.go:2042 +0x44
net/http.(*ServeMux).ServeHTTP(0xc6efc0, 0x9bdd00, 0xc000238000, 0xc000226000)
	/usr/local/go/src/net/http/server.go:2417 +0x1ad
net/http.serverHandler.ServeHTTP(0xc0000b4000, 0x9bdd00, 0xc000238000, 0xc000226000)
	/usr/local/go/src/net/http/server.go:2843 +0xa3
net/http.(*conn).serve(0xc000208000, 0x9bec40, 0xc00021e000)
	/usr/local/go/src/net/http/server.go:1925 +0x8ad
created by net/http.(*Server).Serve
	/usr/local/go/src/net/http/server.go:2969 +0x36c
```
(`127.0.0.1:7080` was previously `mail.<mydomain>.com`, and I changed it in hopes this would fix it)

prometheus.yml
```
  - job_name: 'mailcow'
    static_configs:
      - targets: [ 'localhost:9099' ]
    params:
      host: [ '127.0.0.1:7080' ]
      apiKey: [ 'xxxxxx-xxxxxx-xxxxxx-xxxxxx-xxxxxx' ]
```

docker-compose.override.yml (I'm hosting this alongside mailcow-dockerized, and it was working great)
```
exporter-mailcow:
        image: thej6s/mailcow-exporter
        restart: always
        ports:
         - "127.0.0.1:9099:9099"
```

I needed Grafana to email me, and Gmail was bouncing mail coming from my server -- The rDNS wasn't set up properly. Then, I had some issues with IPv6 and went through enabling that. It was mostly adding "ports: - HTTPS_BIND_IPV6" etc to the docker-compose.override.

I really have no idea where to start with this. Being a C# developer. Please let me know if you need any more info.

A few hours ago:
![chrome_x8IR3YL5mG](https://user-images.githubusercontent.com/10319195/184466463-33233492-67c1-49dd-a45d-cbee71e87c0b.png)
Right now:
![chrome_Giy7NAAhn8](https://user-images.githubusercontent.com/10319195/184466468-2dc0d134-11b4-4ba0-b644-72cd42166680.png)

Only a few metrics are showing, almost none of the ones I want though:
![chrome_PGkneyTOnT](https://user-images.githubusercontent.com/10319195/184466485-00efbfed-7fde-4ed6-9c1c-df1702926f64.png)

**Comment by alexandrepossebom** [comment]
I made a pull request to address this panic message you got above.

BTW do you mind to share your grafana dashboards? I liked it.

**Comment by TcNobo** [comment]
I don't have it anymore after uninstalling and reinstalling, but when I do eventually come back to this I'll share when I recreate it.

**Comment by j6s** [comment]
I really like the dashboard indeed - Looks much more advanced than the one I have setup myself. I am currently on vacation, but will get around to checking the PR sometime next week.

**Comment by j6s** [comment]
Solved by #12 . Thank you @alexandrepossebom
